python/py-pdb2pqr/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               select 1.0
+PortGroup               github 1.0
+name                    py-pdb2pqr
+version                 3.6.1
+revision                0
+
+supported_archs         noarch
+platforms               {darwin any}
+license                 BSD
+maintainers             {gmail.com:howarth.at.macports @jwhowarth} openmaintainer
+
+description             Preparation of protein structures for electrostatics calculations
+long_description        {*}${description}
+
+homepage                https://github.com/Electrostatics/pdb2pqr
+github.setup            Electrostatics    pdb2pqr    ${version} v
+github.tarball_from     releases
+github.tarball_from     archive
+master_sites            https://github.com/Electrostatics/pdb2pqr/archive/refs/tags/v${version}:pdb2pqr
+distfiles               pdb2pqr-v${version}.tar.gz:pdb2pqr
+
+checksums               rmd160  bdd66b7cc6ecf069c709bb5b8eb0665ec42602d2 \
+                        sha256  48ab09877e9c35dad4c46459ad31422013ea153e3681fc2cace5bf68ce352d75 \
+                        size    13389894
+
+python.versions         38 39 310 311 312
+
+patchfiles              skip-network-tests.patch
+patch.pre_args          -p1
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-numpy \
+                        port:py${python.version}-pandas \
+                        port:py${python.version}-pdbx \
+                        port:py${python.version}-propka \
+                        port:py${python.version}-pytest \
+                        port:py${python.version}-requests \
+                        port:py${python.version}-setuptools \
+                        port:py${python.version}-testfixtures
+
+    depends_run-append  port:pdb2pqr_select
+
+    test.run        yes
+    test.cmd        ${python.bin} -m pytest tests
```

Comment:
the default test phase setup by the `python` PortGroup assumes `pytest` and sets `test.cmd` already; this can likley be removed. (Also the explicit `py-pytest` dependency is unneeded, it's already added.